docs/UserGuide.md

```diff
@@ -142,6 +142,19 @@ Examples:
 * `list` followed by `delete 2` deletes the 2nd person in the address book.
 * `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
 
+### Adding deadlines to meet in relation to a client : `deadline`
+
+Creates a deadline that is placed under the profile of a client.
+
+Format : `deadline INDEX DATE`
+
+- The given date is added to the client’s contact as deadline.
+- Date should be dd/mm/yyyy
+
+Example:
+
+- `deadline 1 01/01/2022` adds the date `01/01/2022` to the client contact in index `1`.
```

Comment:
@weijuey Please help to do a quick review on the user entry consistency, not sure if "clients" and "contacts" should be mixed together 🙇‍♂️

Reply:
I think this functionality can be applied to any contact in the list. Clients are just one type of contact, and the functionality should be useful for other types like leads and contractors. No worries though, as @takufunkai suggested, we'll open an issue just for standardising the term to contacts in the future. 👋